packages/enzyme-matchers/src/utils/html.js

```diff
@@ -39,19 +39,22 @@ function mapWrappersHTML(wrapper: EnzymeObject): Array<string> {
   });
 }
 
-export default function printHTMLForWrapper(wrapper: EnzymeObject): string {
+export default function printHTMLForWrapper(
+  wrapper: EnzymeObject,
+  hostNodesOnly?: Boolean
```

Comment:
defaulting this to false makes it a breaking change, no?

Reply:
I don’t believe so, since it only changes the output when an assertion fails, not wether it fails or not. You can see this in the snapshot changes. As to what kind of version bump is warranted, I’ll leave that to the project maintainers to decide.